include/rocksdb/statistics.h

```diff
@@ -468,7 +473,13 @@ class Statistics {
   virtual void recordTick(uint32_t tickerType, uint64_t count = 0) = 0;
   virtual void setTickerCount(uint32_t tickerType, uint64_t count) = 0;
   virtual uint64_t getAndResetTickerCount(uint32_t tickerType) = 0;
-  virtual void measureTime(uint32_t histogramType, uint64_t time) = 0;
+  virtual void measureTime(uint32_t histogramType, uint64_t time) {
+    if (stats_level_ <= StatsLevel::kExceptTimers) {
+      return;
+    }
+    recordInHistogram(histogramType, time);
+  }
+  virtual void recordInHistogram(uint32_t histogramType, uint64_t time) = 0;
```

Comment:
Should we be worried about backward compatibility here?

Reply:
I thought about still keeping measureTime() but I didn't figure out what I should call the new function. measureTimeForReallyTime()? But it's a good point. Let me think about it. Maybe I can come up with something better.